platforms/hyperledger-fabric/configuration/roles/helm_component/templates/install_chaincode_job.tpl

```diff
@@ -39,6 +41,7 @@ spec:
       builder: hyperledger/fabric-ccenv:{{ network.version }}
       name: {{ component_chaincode.name | lower | e }}
       version: {{ component_chaincode.version }}
+      lang: {{ component_chaincode.lang }}
```

Comment:
if this fails when lang does not exist in network.yaml, then that needs to be corrected.